README.md

```diff
@@ -21,9 +16,20 @@
     
 - To run a specific simulation
   
-    mvn clean gatling:test -Dgatling.simulationClass=<simulation class name>
+    mvn clean gatling:test -Dgatling.simulationClass=simulation_class_name
     
```

Comment:
Wondering why <simulation class name> notation is now shortened.  Could we use a ${} notation to indicate that this is a variable that does not need to be hard coded

Reply:
good point I've fixed that. But I've used {} instead of ${} for placeholders not to confuse it with env vars